scripts/release_info.py

```diff
@@ -280,7 +275,7 @@ def get_blocking_bug_issue_count(repo):
         int: Number of unresolved blocking bugs.
     """
     blocking_bugs_milestone = repo.get_milestone(
-        number=BLOCKING_BUG_MILESTONE_NUMBER)
+        number=feconf.BLOCKING_BUG_MILESTONE_NUMBER)
```

Comment:
Perhaps should be BLOCKING_BUGS_MILESTONE_NUMBER (plural)

Also, thinking about it -- could you also verify (in an appropriate location, not necessarily here) that the milestone is currently open/active (and error if it isn't), to guard against the possibility that we close it in the future (and replace it with something different)?

Reply:
I was thinking to create OppiaGithub class (or some different name) sub class of github.Repository.Repository which will have all the GitHub related methods and constants, that way it will be reusable and GitHub related constant will be in a separate place (currently it's inside where we have our app's configurational constants.)

Reply:
I have added a new constants file for release related constants instead.

Reply:
Added the test as well.

Reply:
I have added the test in the corresponding functions itself since we will need the personal access token to fetch the milestone and we cannot ask for that as an input in a separate test.